builds/e2e/longhaul.yaml

```diff
@@ -418,4 +418,69 @@ jobs:
           snitch.storage.account: '$(snitch.storage.account)'
           snitch.storage.masterKey: '$(StorageAccountMasterKeyStress)'
           loadGen.message.frequency: '$(loadGen.message.frequency.amd64)'
+
+################################################################################
+  - job: windows_servercore_amd64_moby
+################################################################################
+    displayName: Windows Server Core AMD64 Moby
+    condition: eq(variables['run.windows.amd64.moby'], 'true')
+    pool:
+      name: $(pool.name)
+      demands:
+        - is-server -equals true
```

Comment:
It is better to define agent.osname which possible values are "Win10Pro", "ServerCore", etc.  This is more feasible to just use a boolean is-server variable.

Reply:
Also you need to add this parameter to Windows amd64 section to distinguish servers to run windows amd64 and windows server core amd64.

Reply:
I saw the vsts agent name is iotedge-server1-longhaul; please use same name as the machine name.

Reply:
Please help to check stress test result by make request (Invoke-WebRequest) to report endpoint, http://<analyzer ip>:15000/api/report; you can get analyzer module ip by running "docker inspect analyzer".

Reply:
@philipktlin Changed up the agent name to match the VM name.

I wanted to use a System variable for targeting but when I compared the System variables of the Server Core agents vs a regular Windows agent, there was no difference. Any advice on what to use here would be great. I'll match whatever variable we use on the Windows config too.

Reply:
Instead of cloning and adding a new variable to control to run on different agent, I am thinking if we can get benefit from using multi-agent option, please read this, https://mitra.computa.asia/articles/msdn-how-execute-agent-phase-multiple-agents-single-vsts-release and try it out.

Reply:
As above comment, I suggest to add a user-defined variable "agent.osname", not system variable.

Reply:
Ahh I understand. I'll give multi-agent a try but I'm not entirely sure how it works. If not, I'll rename the variable to "agent.osname" and also define a similar one for regular Windows.

Reply:
Also, I got the status report for the stress test, and it came back with an OK result.

Reply:
Fixed the name of the variables, and also changed the Release Label to make Server Core machines unique from regular Windows.

Wrt multi-agent testing, this needs more investigation and prototyping effort which is more time than we have right now. We need to hook up the tests and show that they work for now, so I'm going with individually targeting the Server Core VMs. This might be the way forward, so we'll revisit this after GA.